docs/organizations/integrations/slack-integration.md

```diff
@@ -0,0 +1,25 @@
+# Organization Slack integration for Security issues
+
+This integration sends notifications to a Slack channel of your choice whenever Codacy detects new critical Security issues in any repository within your organization.
+
+## Installing the Slack integration
+
+To install the Slack integration:
+
+1.  Follow the [instructions on the Slack documentation pages](https://api.slack.com/messaging/webhooks) to add an Incoming Webhook to your Slack workspace and set it to post to a channel of your choice, and then copy the webhook URL.
+
+    The Incoming Webhook URL starts with `https://hooks.slack.com/services/`.
+
+1.  Open your organization **Integrations**, page **Slack**, paste the Incoming Webhook URL in the field, and click **Install Slack**.
+
+    ![Slack integration installation](images/slack-integration-install.png)
+
+Once the Slack integration is installed, Codacy sends a confirmation message to the Slack channel you configured when creating the Incoming Webhook.
+
+## Uninstalling the Slack integration
```

Comment:
Before describing the uninstall use case, I miss the use case showing the value of the integration... 🤔 
After installing the integration, what can the user do/see?

Reply:
OK, I'm extending the paragraph above to reiterate the main use case, plus adding a screenshot as suggested by @mrfyda 👍